src/module.ts

```diff
@@ -38,23 +38,21 @@ export default defineNuxtModule<ModuleOptions>({
   setup(options, nuxt) {
     const logger = useLogger('turnstile')
     const siteKey = options.siteKey || nuxt.options.runtimeConfig.public?.turnstile?.siteKey
+
     if (!siteKey) {
       logger.warn(
         'No site key was provided. Make sure you pass one at runtime by setting NUXT_PUBLIC_TURNSTILE_SITE_KEY.'
       )
     }
 
-    if (options.secretKeyPath) {
-      try {
-        options.secretKey = fs.readFileSync(
-          resolve(nuxt.options.rootDir, options.secretKeyPath),
-          'utf-8'
-        )
-      } catch {}
```

Comment:
The try-catch looks awkward, should we keep it? Or is there any better style to write such a thing?

Reply:
we could use async with an inline `.catch(() => null)`